mode-apis-base/src/alertingModels.ts

```diff
@@ -226,53 +226,40 @@ export enum AlertState {
     INEFFECTIVE = 4,
 }
 
+export interface Alert {
```

Comment:
Should we add `condition` to the base Alert interface too?
```
readonly condition: AlertCondition;
```

Reply:
Ah, that's better, I will do that.

Reply:
I did